<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>repository</class>
    <widget class="QWidget" name="repository">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>936</width>
                <height>643</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <property name="bottomMargin">
                        <number>0</number>
                    </property>
                    <item>
                        <widget class="QLabel" name="repoLabel">
                            <property name="text">
                                <string>repo</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="watchButton">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Watch</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="forkButton">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Fork</string>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QPushButton" name="starButton">
                            <property name="sizePolicy">
                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                    <horstretch>0</horstretch>
                                    <verstretch>0</verstretch>
                                </sizepolicy>
                            </property>
                            <property name="text">
                                <string>Star</string>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
            <item>
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="code">
                        <attribute name="title">
                            <string>Code</string>
                        </attribute>
                        <layout class="QHBoxLayout" name="horizontalLayout">
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_3">
                                            <item>
                                                <widget class="QComboBox" name="branchComboBox">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QWidget" name="branchWidget" native="true">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QWidget" name="tagsWidget" native="true">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <spacer name="horizontalSpacer_2">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>40</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="codeButton">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="text">
                                                        <string>Code</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QFrame" name="frame">
                                            <property name="frameShape">
                                                <enum>QFrame::StyledPanel</enum>
                                            </property>
                                            <property name="frameShadow">
                                                <enum>QFrame::Plain</enum>
                                            </property>
                                            <layout class="QHBoxLayout" name="horizontalLayout_4">
                                                <item>
                                                    <widget class="QLabel" name="profileImageLabel">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="text">
                                                            <string>img</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="usernameLabel">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="text">
                                                            <string>username</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="commitLabel">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="text">
                                                            <string>commit desc</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <spacer name="horizontalSpacer_3">
                                                        <property name="orientation">
                                                            <enum>Qt::Horizontal</enum>
                                                        </property>
                                                        <property name="sizeHint" stdset="0">
                                                            <size>
                                                                <width>40</width>
                                                                <height>20</height>
                                                            </size>
                                                        </property>
                                                    </spacer>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="numsLabel">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="text">
                                                            <string>nums</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="timeLabel">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="text">
                                                            <string>1 day ago</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="IconLabel" name="commitsWidget" native="true">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QListWidget" name="filesListWidget"/>
                                    </item>
                                </layout>
                            </item>
                            <item>
                                <layout class="QVBoxLayout" name="verticalLayout_3">
                                    <property name="sizeConstraint">
                                        <enum>QLayout::SetFixedSize</enum>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_2">
                                            <property name="sizeConstraint">
                                                <enum>QLayout::SetFixedSize</enum>
                                            </property>
                                            <item>
                                                <widget class="QLabel" name="label">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="font">
                                                        <font>
                                                            <bold>true</bold>
                                                        </font>
                                                    </property>
                                                    <property name="text">
                                                        <string>About</string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <spacer name="horizontalSpacer">
                                                    <property name="orientation">
                                                        <enum>Qt::Horizontal</enum>
                                                    </property>
                                                    <property name="sizeType">
                                                        <enum>QSizePolicy::Fixed</enum>
                                                    </property>
                                                    <property name="sizeHint" stdset="0">
                                                        <size>
                                                            <width>80</width>
                                                            <height>20</height>
                                                        </size>
                                                    </property>
                                                </spacer>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="configButton">
                                                    <property name="sizePolicy">
                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                            <horstretch>0</horstretch>
                                                            <verstretch>0</verstretch>
                                                        </sizepolicy>
                                                    </property>
                                                    <property name="text">
                                                        <string/>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="descLabel">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>desc</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="IconLabel" name="licenseWidget" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="IconLabel" name="starWidget" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="IconLabel" name="watchingWidget" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="IconLabel" name="forksWidget" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="Line" name="line">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="label_2">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="font">
                                                <font>
                                                    <bold>true</bold>
                                                </font>
                                            </property>
                                            <property name="text">
                                                <string>Releases</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="IconLabel" name="releasesWidget" native="true">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="verticalSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Vertical</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>20</width>
                                                    <height>40</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                </layout>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="issues">
                        <attribute name="title">
                            <string>Issues</string>
                        </attribute>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>IconLabel</class>
            <extends>QWidget</extends>
            <header>widgets.IconLabel</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
